src/taipy/rest/api/resources/datanode.py

```diff
@@ -55,8 +55,11 @@ class DataNodeResource(Resource):
     get:
       tags:
         - api
-      summary: Get a datanode
-      description: Get a single datanode by ID
+      summary: Get a data node
+      description: |
+        Return a single data node by DataNodeId. If the data node does not exist, a 404 error is returned.
```

Comment:
"Return a single data node by DataNodeId."
->
"Returns the data node identified by *datanode_id*."

(and similar phrasings in all the files)